packages/aws-cdk-lib/aws-eks/lib/cluster.ts

```diff
@@ -1666,6 +1680,74 @@ export class Cluster extends ClusterBase {
       throw new Error('Cannot specify serviceIpv4Cidr with ipFamily equal to IpFamily.IP_V6');
     }
 
+    if (props.remoteNodeNetworks) {
```

Comment:
Line 1683 to Line 1749 are for validations and the validations are quite long, I'd prefer to move it out a separate validation function.

Reply:
Done thanks!